doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -190,15 +190,60 @@ In a worst case scenario, the store time can be up to several minutes.
 This can for example happen when storing a large size replay protection list.
 It is recommended to configure the settings subsystem's internal caches to improve the performance.
 
-* The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
+NVS lookup cache reduces the number of search loops within NVS' application table.
 
-  * :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE`.
+* :kconfig:option:`CONFIG_NVS_LOOKUP_CACHE`.
 
-* The size of the Settings NVS name cache is recommended to be set to the maximum number of devices the configured device communicates with.
-  For example, if a device communicates with 255 other devices, it is worth setting the cache to minimum 255.
+The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
 
-  * :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE_SIZE`.
+* :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE`.
 
-* NVS lookup cache reduces the number of search loops within NVS' application table.
+The size of the Settings NVS name cache, :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE_SIZE`, is recommended to be at least equal to the number of settings entries the device is expected to store.
 
-  * :kconfig:option:`CONFIG_NVS_LOOKUP_CACHE`.
+The Bluetooth Mesh stack stores the following data persistently:
+
+* Network information (primary address and device key)
+* Configuration parameters (supported features, default TTL, network transmit- and relay retransmit parameters)
```

Comment:
Why hyphen in network transmit-?

Reply:
The hyphen was included to not write "parameters" twice. I. e. instead of `network transmit parameters and relay retransmit parameters`

Reply:
The same goes for the other cases regarding "models". Do you still think I should remove the hyphen?

Reply:
Yeah, I think we can remove these :) You would usually use it with compounds or hyphenated terms that are listed, not necessary here. 
...network transmit and relay retransmit parameters, is ok without the hyphen.
The same goes for the models.

Reply:
Removed hyphens and added changelog entry :)